app/assets/javascripts/redux/modules/models/listenSignals.js

```diff
@@ -168,3 +190,16 @@ export const addListenSignalData = (payload) => {
     ],
   });
 };
+
+export const updateListenSignalData = (payload, signalId) => {
+  return createRequestAction({
+    endpoint: listenSignalEndpoint(signalId),
+    method: 'PUT',
+    body: JSON.stringify(payload),
+    types: [
+      { type: LISTEN_SIGNALS_PUT_REQUEST, meta: { signalId } },
```

Comment:
On a failure case for a `put` request, we're not going to be getting the resource id back from the response. We can use the meta prop to set the error on the right listen signal if we so choose. My point is that if we pass `signalId` as meta prop, [`meta.id` here](https://github.com/project-signal/project-signal/blob/post-request/app/assets/javascripts/redux/modules/models/listenSignals.js#L72) will be undefined.